unitTests/ui/MissingTermTest.ts

```diff
@@ -0,0 +1,69 @@
+import * as Mock from '../MockEnvironment';
+import { MissingTerm } from '../../src/ui/MissingTerm/MissingTerm';
+import { IMissingTermOptions } from '../../src/ui/MissingTerm/MissingTerm';
+import { $$ } from '../../src/Core';
+import { FakeResults } from '../Fake';
+
+export function MissingTermTest() {
+  describe('MissingTerm', () => {
+    let test: Mock.IBasicComponentSetup<MissingTerm>;
+    describe('exposes options', () => {
+      let fakeResult = FakeResults.createFakeResult();
+
+      beforeEach(() => {
+        fakeResult['absentTerms'] = ['will'];
+        fakeResult.state.q = 'This will be a really long query';
+      });
+
+      describe('clickable when set to', () => {
```

Comment:
I find this description hard to understand without reading the `it` blocks. What do you think of: `when the 'clickable' option is set to`.

Reply:
you're right, I'll go with your suggestion, it's a lot easier to read